test/games/strategy/triplea/ui/TestRoute.java

```diff
@@ -71,7 +71,7 @@ public void testPointSplitting(){
   
   @Test
   public void testPointAcquisition(){
-    assertArrayEquals(dummyPoints, spyRouteDrawer.getRoutePoints(dummyRouteDescription, dummyMapData));
+    assertArrayEquals(dummyPoints, spyRouteDrawer.getRoutePoints(dummyRouteDescription, dummyMapData, 0, 0, 0, 0));
```

Comment:
Code smell here - each parameter should impact the outcome of a test. Otherwise, it indicates that there are multiple concerns being addressed with the single method. Often when this happens the method should be broken up into two different methods.